### ISSUE TITLE
Emails not being sent but system emails work fine

### ISSUE THREAD

**Issue by dbrandesky** [freescout-helpdesk/freescout#1562]
My system email account (for sending new user links and test emails) and my primary mailbox (for actual user support tasks) share the same email address: support@example.com. If I go to Manage > Settings > Mail Settings and send a test email using SMTP, the email gets sent, logged as sent, and I see it in the inbox of the account to which I sent it. However, if I go to my actual support mailbox and attempt to send emails, I get nothing - no log, no message in my support email's outbox or sent folder, and no email received on the other end. It's like it's not even attempting to send emails.

**Comment by freescout-helpdesk** [comment]
https://github.com/freescout-helpdesk/freescout/wiki/Sending-Issues

**Comment by dbrandesky** [comment]
> https://github.com/freescout-helpdesk/freescout/wiki/Sending-Issues

Following the steps in the linked article:

1. "Sending Emails" Log: The only entries I see shown are of type "Test Email". These are the emails that were actually delivered to the appropriate destination. Emails sent directly to a customer from the mailbox were not delivered.
2. "Outgoing Emails" in conversation thread: This option doesn't seem to exist? All I see are "Follow", "Forward", "Print", and "Merge" under the ellipsis (...) in a conversation.
3. PHP modules: everything reports "OK" in Manage > System > PHP. `php -m` reports the following:

`root@freescout:~# php -m
[PHP Modules]
calendar
Core
ctype
curl
date
dom
exif
FFI
fileinfo
filter
ftp
gd
gettext
hash
iconv
imap
json
libxml
mbstring
mysqli
mysqlnd
openssl
pcntl
pcre
PDO
pdo_mysql
Phar
posix
readline
Reflection
session
shmop
SimpleXML
sockets
sodium
SPL
standard
sysvmsg
sysvsem
sysvshm
tokenizer
xml
xmlreader
xmlwriter
xsl
Zend OPcache
zip
zlib

[Zend Modules]
Zend OPcache`

4. I am using SMTP, but I'm not sure where to find the SMTP server logs? I am going to inquire with my email service provider, but the admin console doesn't report any bounced, undeliverable, or otherwise rejected emails.
5. Background jobs: **This appeared to be the issue.** The cron task was enabled, but the queue:work daemon was not running; I found this all the way at the bottom of the Manage > System screen. "queue:work" had a last successful run of "?", and it recommended running the "Clear Cache" command. Conveniently, there was a hyperlink to run "Clear Cache" from the web UI. Now I have a boat load of unread emails I sent to my test account!

I will continue to monitor the situation to see if I have more problems, I'm not quite sure where all the instability occurred but it seemed to have something to do with installing modules; after I installed the first one, I had to manually install the subsequent ones, which required some other backend commands to get things going.
